### DISCUSSION TITLE
Put icons back on the left (or make it an option)

### DISCUSSION THREAD

**Discussion by darth-hp** [ViennaRSS/vienna-rss#1429] [Ideas]
I don't know if it's big-sur or a recent Vienna Update but icons are now located on the right. This is very inefficient moving around because I pick entries on the left, move down to some entries, move up and right to some icon, again left and so forth. Before it was mostly a little bit up and down and only a little left and right.
As a work around I have put tons of spaces in the bar to get them back to the old location.
<img width="560" alt="image" src="https://user-images.githubusercontent.com/13313834/104282047-b99c4100-54ae-11eb-98bd-427c3485a928.png">

**Comment by Eitot** [comment]
This is indeed a change in Big Sur; it is how the default toolbar style works.

 I happen to know that you can force the old toolbar style with this command:
```
defaults write uk.co.opencommunity.vienna2 NSWindowSupportsAutomaticInlineTitle -bool false
```

**Reply by josh64x2** [reply]
We could always implement a preference if others are interested. I believe it is covered in detail at https://youtu.be/Ri6ou6ridac

**Comment by darth-hp** [comment]
That would be awesome
